HealthcareSystem/PatientWebApp/wwwroot/js/activate_account.js

```diff
@@ -46,4 +48,34 @@
     });
     });
     
-});
\ No newline at end of file
+});
+
+function encryptJmbg(jmbg) {
```

Comment:
Wouldn't this be decrypt jmbg?

Reply:
Okay, I'll consider moving the decryption to the backend code.